### ISSUE TITLE
Format data for BasicHtmlWebResponseObject contains ParsedHtml and Forms properties

### ISSUE THREAD

**Issue by markekraus** [PowerShell/PowerShell#5373]
PowerShell Core (currently) only supports basic parsing on Web Cmdlets. All of the code which contains the actual `Forms` and `ParsedHtml`properties are behind `#if !CORECLR` so these properties do not exist on any of the core types. However, The `Forms` and `ParsedHtml` properties are defined in the format data.

I believe [the decision](https://github.com/PowerShell/PowerShell/issues/3267#issuecomment-286917402) was that HTML processing would no longer be handled by the `WebResponseObject` and instead would be part of a separate cmdlet. 

Over the course of the past few months have fielded a lot of confused questions about this. I think it would help clarify that this is not supported in Core if the formatting data was updated to match object and remove the dummy `Forms` and `ParsedHtml` properties it adds.

Offending code

https://github.com/PowerShell/PowerShell/blob/fe3e44f3055ccd57e19ce1d29a5320e2f3891705/src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs#L1263

https://github.com/PowerShell/PowerShell/blob/fe3e44f3055ccd57e19ce1d29a5320e2f3891705/src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs#L1268

Steps to reproduce
------------------

```powershell
$result = iwr 'https://www.bing.com/'
$result
```

Expected behavior
-----------------
`Forms` and `ParsedHtml` should not be shown
```none
StatusCode        : 200
StatusDescription : OK
Content           : <!DOCTYPE html PUBLIC #snip
RawContent        : HTTP/1.1 200 OK #snip
Headers           : {[Cache-Control, System.String[]], #snip
Images            : {@{outerHTML=<img id= #snip
InputFields       : {@{outerHTML=<input clas #snip
RawContentLength  : 124089
RelationLink      : {}
```

Actual behavior
---------------
`Forms` and `ParsedHtml` are shown
```none
StatusCode        : 200
StatusDescription : OK
Content           : <!DOCTYPE html PUBLIC #snip
RawContent        : HTTP/1.1 200 OK #snip
Forms             :
Headers           : {[Cache-Control, System.String[]], #snip
Images            : {@{outerHTML=<img id= #snip
InputFields       : {@{outerHTML=<input clas #snip
ParsedHtml        :
RawContentLength  : 124089
RelationLink      : {}
```

Environment data
----------------

```powershell
Name                           Value
----                           -----
PSVersion                      6.0.0-beta.9
PSEdition                      Core
GitCommitId                    v6.0.0-beta.9
OS                             Microsoft Windows 10.0.15063
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by SteveL-MSFT** [comment]
Agree we should remove those properties.

**Comment by markekraus** [comment]
@SteveL-MSFT  Currently the formatting data is lumped together for both `BasicHtmlWebResponseObject` (which exists in Core) and `HtmlWebResponseObject` (which does not exist in core). Should this be just removed for both, or have them separated in 2 different formats, or kill off the formatting for `HtmlWebResponseObject` and remove these 2 properties for `BasicHtmlWebResponseObject`?

**Comment by SteveL-MSFT** [comment]
Since we've already made the decision to clean up the PowerShell Core code base to not have FullClr code, I would actually like to clean up the entirety of the webcmdlets code to just have CoreClr code (so we don't have to worry about the question you are asking).

**Comment by markekraus** [comment]
@SteveL-MSFT Is that cleanup something that can be done before release, though? Ideally, we would at least remove these 2 properties before release, But if it is tied to a bigger cleanup, maybe not?

**Comment by SteveL-MSFT** [comment]
@markekraus the clean-up is not needed for 6.0.0 and agree we can wait on that.  Removing the 2 properties we should get in for RC so it doesn't become a breaking change (from 6.0.0) later.

**Comment by markekraus** [comment]
@SteveL-MSFT I removed that properties in #5376 with the other cleanup activities. Would that one get added before RC? If not, I can separate it out to a PR of its own.

**Comment by SteveL-MSFT** [comment]
@markekraus I think the PR can still make it into RC as-is
